nixos/modules/services/misc/mpdscribble.nix

```diff
@@ -0,0 +1,292 @@
+pidfile = ${cfg.pidfile}
+
+# Change to this system user after daemonization.
+daemon_user = ${cfg.daemonUser}
+
+# The location of the mpdscribble log file.  The special value
+# "syslog" makes mpdscribble use the local syslog daemon.  On most
+# systems, log messages will appear in /var/log/daemon.log then.
+# "-" means log to stderr (the current terminal).
+log = ${cfg.logFile}
+
+# How verbose mpdscribble's logging should be.  Default is 1.
+verbose = ${cfg.verbose}
+
+# How often should mpdscribble save the journal file? [seconds]
+journal_interval = ${cfg.journalInterval}
+
+# The host running MPD, possibly protected by a password
+# ([PASSWORD@]HOSTNAME).  Defaults to $MPD_HOST or localhost.
+host = ${cfg.host}
+
+# The port that the MPD listens on and mpdscribble should try to
+# connect to.  Defaults to $MPD_PORT or 6600.
+port = ${cfg.port}
+
+${mkSection cfg "lastfm" "last.fm"}
+${mkSection cfg "librefm" "libre.fm"}
+${mkSection cfg "jamendo" "jamendo"}
+'';
+in
+
+{
+  ###### interface
+
+  options = {
+
+    services.mpdscribble = {
+
+      enable = mkOption {
+        default = false;
+        description = "
+          Whether to enable mpdscribble.
+        ";
+      };
+
+      proxy = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        description = ''
+          HTTP proxy URL.
+        '';
+      }
+
+      pidfile = mkOption {
+        default = "/var/run/mpdscribble.pid";
+        type = types.path;
+        description = ''
+          The location of the pid file. mpdscribble saves its process id there.
+        '';
+      }
+
+      daemonUser = mkOption {
+        default = "mpdscribble";
+        description = ''
+          Change to this system user after daemonization.
+        '';
+      }
+
+      logFile = mkOption {
+        default = "/dev/null";
+        example = "/tmp/mpdscribble.log";
+        description = ''
+          Location of the logfile.
+        '';
+      };
+
+      verbose = mkOption {
+        default = 1;
+        type = types.int;
+        description = ''
+          Log level for the mpdscribble daemon.
+        '';
+      };
+
+      journalInterval = mkOption {
+        default = 600;
+        example = 60;
+        type = types.int;
+        description = ''
+          How often should mpdscribble save the journal file? [seconds]
+        '';
+      };
+
+      host = mkOption {
+        default = "localhost";
+        type = types.str;
+        description = ''
+          Host for the mpdscribble daemon to search for a mpd daemon on.
+        '';
+      };
+
+      port = mkOption {
+        default = "localhost";
+        type = types.int;
+        description = ''
+          Port for the mpdscribble daemon to search for a mpd daemon on.
+        '';
+      };
+
+      lastfm = {
+        enable = mkOption {
+          default = false;
+          type = types.bool;
+          description = "
```

Comment:
Better use '' (two single quotes for multiline string).
But this is only one line so you can reduce two "...." in one line.